contracts/sol6/Dao/KyberStaking.sol

```diff
@@ -22,11 +22,11 @@ contract KyberStaking is IKyberStaking, EpochUtils, ReentrancyGuard {
     IERC20 public kncToken;
```

Comment:
Lets wait for final names for StakerData struct
before merging.

Reply:
Changing name should be in logic PR?

Reply:
yes lets change in logic PR
since the new function added also uses same names.